fusor-ember-cli/app/templates/subscriptions/management-application.hbs

```diff
@@ -16,6 +16,19 @@
       {{loading-spinner show=true text=msgWaiting}}
 {{/if}}
 
+{{#if showErrorMessage}}
+  <div class="row">
+    <div class='col-md-9'>
+      <div class='alert alert-danger rhci-alert'>
+          <i class="fa fa-2x fa-exclamation-triangle errorForValidation"></i>
+          &nbsp;
+          {{newSatelliteName}} failed to be added. Check the management application name and try again.
```

Comment:
@dymurray, you may want to make this generious `{{errorMsg}}` unless this is the only possible error on this page.